src/test/java/teammates/test/pageobjects/InstructorCourseEditPage.java

```diff
@@ -162,6 +184,23 @@ public void verifyInstructorEditFormDisabled(int instrNum) {
         assertTrue(isNotEditable);
     }
 
+    public void verifyInstructorChangesAfterEdit(int instrNum, String newName, String newEmail,
```

Comment:
Renamed in https://github.com/TEAMMATES/teammates/pull/7559/commits/f2fb267d58bef77075d037a9ede2db1368a1b08a and added documentation in https://github.com/TEAMMATES/teammates/pull/7559/commits/8774251e0c0e4f68163cbbb70aa61b371e9fc51b